internal/command/context.go

```diff
@@ -22,6 +22,9 @@ const (
 	ArchArm64 Architecture = "arm64"
 	// ArchArm represents the arm architecture
 	ArchArm Architecture = "arm"
+	// ArchMultiple represents the universal architecture used by some OS to
+	// identify a binary that supports multiple architectures (fat binary)
+	ArchMultiple Architecture = "multiple"
```

Comment:
How about calling this "universal"?

Reply:
it was the initial proposal, but we changed to multiple architecture to align with the android docs as pointed out by Andrew. 
Probably if we want to uniform to the "universal" term across android and iOS / macOS this is a good time :)

Reply:
I'm not hugely fussed either way - it was mostly that the docs need to be clear, the concept of universal varies across platforms and versions so we need to be careful how it is described.
Universal, in my mind at least, means "all architectures" which may or not be what is built under the new feature.